src/index.ts

```diff
@@ -1,10 +1,11 @@
 import OAuth from './OAuth';
 import RestClient from './RestClient';
+import * as types from './types';
 
 const BigCommerce = {
   Auth: OAuth,
   Rest: RestClient,
+  Types: types,
 };
 
-export default BigCommerce;
-export * from './types';
+export = BigCommerce;
```

Comment:
do we need to set a variable name here? @matthewvolk

Reply:
No, the equal sign is important here. More info: https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require

> TypeScript supports `export =` to model the traditional CommonJS and AMD workflow.